packages/@react-native/repo-config/overrides.json

```diff
@@ -1,11 +1,11 @@
 {
-  "baseVersion": "0.0.0-12543d557",
+  "baseVersion": "0.0.0-9b973f32e",
   "overrides": [
     {
-      "type": "derived",
+      "type": "copy",
```

Comment:
Previously patched out a line pulling in a file-version of eslint config that is now more permissive.